packages/@aws-cdk/aws-pipes-alpha/lib/logs.ts

```diff
@@ -36,6 +39,64 @@ export enum LogLevel {
   TRACE = 'TRACE',
 }
 
+/**
+ * Log format for `S3LogDestination` logging configuration.
+ *
+ * @see https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-properties-pipes-pipe-s3logdestination.html#cfn-pipes-pipe-s3logdestination-outputformat
+ */
+export enum S3OutputFormat {
+  /**
+  * Plain text
+  */
+  PLAIN = 'plain',
+  /**
+  * JSON
+  */
+  JSON = 'json',
+  /**
+  * W3C extended log file format
+  * @see https://www.w3.org/TR/WD-logfile
+  */
+  W3C = 'w3c',
+}
+
+/**
+ * Properties for `S3LogDestination`.
+ */
+export interface S3LogDestinationProps {
+  /**
+   * The S3 bucket to deliver the log records for the pipe.
+   *
+   * The bucket can be in the same or a different AWS Account. If the bucket is in
+   * a different acccount, specify `bucketOwner`. You must also allow access to the
+   * Pipes role in the bucket policy of the cross-account bucket.
+   *
+   * @see https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-properties-pipes-pipe-s3logdestination.html#cfn-pipes-pipe-s3logdestination-bucketname
+   */
+  readonly bucket: IBucket;
+  /**
+   * The AWS Account that owns the Amazon S3 bucket to which EventBridge delivers the log records for the pipe.
+   *
+   * @see https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-properties-pipes-pipe-s3logdestination.html#cfn-pipes-pipe-s3logdestination-bucketowner
+   * @default - account ID derived from `bucket`
+   */
+  readonly bucketOwner?: string;
+  /**
+   * The format for the log records.
+   *
+   * @see https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-properties-pipes-pipe-s3logdestination.html#cfn-pipes-pipe-s3logdestination-outputformat
+   * @default `S3OutputFormat.PLAIN`
```

Comment:
Considering the linked documentation says that the only supported format is `json`, I think that would be a more sensible default here.

Reply:
Good shout. I removed this from the integration test and the default is indeed `json`.